RocketChat/Rocket.Chat.ReactNative
# feat: quotes on E2EE Messages

## Proposed changes


This improvement was introduced on the web with this PR https://github.com/RocketChat/Rocket.Chat/pull/26303


## Issue(s)	

Closes https://github.com/RocketChat/Rocket.Chat.ReactNative/issues/4732

## How to test or reproduce


- Enter your E2EE password
- Create or join an encrypted channel
- Quote an older encrypted message
- You should see the `key` icon on the message that will be quoted and on the quote. too

## Screenshots

#### Before
<img width="349" alt="Screen Shot 2023-02-02 at 00 05 11" src="https://user-images.githubusercontent.com/47038980/216221396-5da3d707-7681-4efa-8d3b-f1b1b85ca795.png">


#### After
![Screen Shot 2023-02-01 at 20 12 06](https://user-images.githubusercontent.com/47038980/216187864-342ab254-5dcc-4b71-b29d-9d06b283cf21.png)


## Types of changes




## Checklist



## Further comments

[TC-311]

[TC-311]: https://rocketchat.atlassian.net/browse/TC-311?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
